OpenRA.Game/Player.cs

```diff
@@ -24,7 +24,14 @@
 
 namespace OpenRA
 {
-	public enum PowerState { Normal, Low, Critical }
+	[Flags]
+	public enum PowerState
+	{
+		Normal = 0,
```

Comment:
This appears to have been the problem, if I change them to 1, 2, 4 respectively, all problems are gone.